# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/xorg-server-backtrace.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./hw/xfree86/common/xf86Events.c.orig     2000-01-01 00:00:00.000000000 +0100
+++ ./hw/xfree86/common/xf86Events.c  2005-09-27 16:11:15.000000000 +0200
@@ -1285,6 +1285,10 @@
 # endif
 #endif

+#if defined(__UCLIBC__)
+#  undef  HAVE_BACKTRACE
+#endif
+
 #ifdef HAVE_BACKTRACE
 #include <execinfo.h>

